core/src/main/java/org/apache/accumulo/core/client/admin/InstanceOperations.java

```diff
@@ -149,6 +150,46 @@ boolean testClassLoad(final String className, final String asTypeName)
    */
   void waitForBalance() throws AccumuloException;
 
+  /**
+   * Throws an exception if a tablet server can not be contacted.
```

Comment:
These javadocs should say what the method does normally. The explanation for when an exception is thrown can be in a javadoc `@throws SomeException when <explanation here>` line.

Reply:
This was just a mistake by me. I pushed those up without editing the description of them.